packages/checkbox/src/uxp-checkbox.css

```diff
@@ -19,4 +19,12 @@ governing permissions and limitations under the License.
 
 #input {
     opacity: 0;
+    width: 100%;
+    height: 100%;
 }
+
+#label {
+    margin-top: var(--mod-checkbox-top-to-text,var(--spectrum-checkbox-top-to-text) );
+    margin-left: var(--mod-checkbox-text-to-control,var(--spectrum-checkbox-text-to-control) );
+}
+
```

Comment:
Can you check?, we have to handle and circumvent logical property for id="box" in the sp-checkbox 

`#box {
height: var(
--mod-checkbox-control-size,
var(—spectrum-checkbox-control-size)
);
width: var(
--mod-checkbox-control-size,
var(—spectrum-checkbox-control-size)
);
}
`

Also on host we might have to polyfil the missing logical property pertaining
```suggestion

```

`:host {
max-width: 100%;
min-height: var(—mod-checkbox-height, var(—spectrum-checkbox-height));
}
`